mautic/mautic
# Reset plain text after clone

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
We had the issue with customers who clone emails with plaintext and then wondered why they see some other links in the tracking URLs table. We decided to reset it on Clone. 
Need note: plain text is generated automatically If not exist during the send then It does not decrease send deliverability



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create an email and add plain text
3. Clone the email
4. Check If the plain text was reset
